packages/remote-config/index.ts

```diff
@@ -38,6 +38,7 @@ import {
 // Note these debug options are not part of a documented, supported API and can change at any time.
 // Consolidates debug options for easier discovery.
 // Uses transient variables on window to avoid lingering state causing panic.
+// eslint-disable-next-line @typescript-eslint/no-unused-vars
```

Comment:
Thinking out loud: this var [is used](https://github.com/firebase/firebase-js-sdk/blob/0131e1fcade96853997fc04ba1fee0e8d40ed445/packages/remote-config/src/client/rest_client.ts#L76), so it's strange lint would think otherwise. I wonder if there's another, preferred way to declare vars on `window`🤔